RELEASE/scripts/autoscend/paths/avatar_of_jarlsberg.ash

```diff
@@ -74,11 +74,14 @@ void jarlsberg_buySkills() //Not certain of Skill Priority Order. Current is a g
 		return;
 	}
 	if(my_level() <= get_property("_auto_jarlsbergSkills").to_int())
+	{
+		return;
+		
+	}
+	if(get_property("_auto_completedJarlsbergSkillTree").to_boolean()) //Prevent us from running through the full list of skills checks more than once per day if we already have all skills
```

Comment:
1. get_property does not do a server hit, it checks mafia internal data
2. the line immediately above already limits it to only checking once per levelup. so this function is only processed 13 times per ascension + (daycount - 1)

Reply:
this applies also to line 117